### PR TITLE
Update Node.js in check workflow to version 18

### PR DESCRIPTION
**Pull request by mgrabovsky**
Follow-up to 920d166.

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
I wonder how to add a link to:

https://github.com/fedora-ci/ciboard-server/blob/main/Dockerfile#L2

Is it good to add a comment that this line should stays in sync with ^ ?

**Comment by mgrabovsky** [comment]
I've added a comment.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -9,7 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14.x]
+        # NOTE: The version we use to build the container image should always be
+        # included here. (See Dockerfile in the root directory.)
```
Comment (Andrei-Stepanov) [inline .github/workflows/build.yaml:13]: In server repo, but OK.

**Review by Andrei-Stepanov — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (Andrei-Stepanov) [reply .github/workflows/build.yaml:13]: You can merge or update commit. Up to you.

**Comment by mgrabovsky** [comment]
Updated and merged.

**Pull request merged**